pyproject.toml

```diff
@@ -10,3 +10,27 @@ exclude = '''
   | win/rl/nethack_rl
 )/
 '''
+
+[tool.isort]
+force_single_line = true
```

Comment:
ah.... I'm not _certain_ this is the best idea, but I'm certainly happy to try it.  If it gets unworkable its easy to revert

Reply:
I don't like that for `__init__.py` files which is why excluded those. But otherwise I like this better, and it's demanded by https://google.github.io/styleguide/pyguide.html#313-imports-formatting